From 0da2e9c215f882ea4c331ac4f17738913beb7404 Mon Sep 17 00:00:00 2001
From: Andreas Schneider <asn@samba.org>
Date: Mon, 14 Jan 2019 11:59:59 +0100
Subject: [PATCH 21/27] lib:tevent: Use correct C99 initializer for tevent_req

Signed-off-by: Andreas Schneider <asn@samba.org>
Reviewed-by: Douglas Bagnall <douglas.bagnall@catalyst.net.nz>
Signed-off-by: root <root@localhost.localdomain>
---
 tevent_req.c | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/tevent_req.c b/tevent_req.c
index 76e27b8f7e9..7821d9ae734 100644
--- a/tevent_req.c
+++ b/tevent_req.c
@@ -84,10 +84,12 @@ struct tevent_req *_tevent_req_create(TALLOC_CTX *mem_ctx,
 	}
 
 	*req = (struct tevent_req) {
-		.internal.private_type		= type,
-		.internal.create_location	= location,
-		.internal.state			= TEVENT_REQ_IN_PROGRESS,
-		.internal.trigger		= tevent_create_immediate(req)
+		.internal = {
+			.private_type		= type,
+			.create_location	= location,
+			.state			= TEVENT_REQ_IN_PROGRESS,
+			.trigger		= tevent_create_immediate(req),
+		},
 	};
 
 	data = talloc_zero_size(req, data_size);
-- 
2.19.1

